src/utils/helpers.ts

```diff
@@ -1,3 +1,5 @@
 import { UserRole } from 'types';
 
 export const isNewbie = (role: UserRole) => role === UserRole.Newbie;
+export const getProgressInPercentages = (progress: number) =>
```

Comment:
Can you update the Carrousel component to use this helper also?

Reply:
Sure 😀 It's done here: 011776014a17ec1e19b5c7300c1ef81750cf57d4 🔥
I've also added:
1) missing photo to the carousel
2) Before this percentages wasn't rounded. 

Now it looks like that:
![image](https://user-images.githubusercontent.com/14184100/68961082-c6050b00-07d1-11ea-9bbf-97168117577b.png)